resources/js/views/Detail.vue

```diff
@@ -1,9 +1,12 @@
 <template>
-    <div class="bg-gray-softest h-48 border-black border-t-2 border-b-2 relative" v-if="item">
+    <div class="bg-gray-softest aspect-w-4 aspect-h-3 border-black border-t-2 border-b-2 relative" v-if="item">
         <ItemImageMovable v-if="$route.meta.edit" :item="item"></ItemImageMovable>
         <ItemImageLightbox v-else :item="item"></ItemImageLightbox>
     </div>
-    <div class="h-full border-black p-4 pb-24" v-if="item">
+    <div class="h-full border-black px-4 pb-24 pt-8 relative" v-if="item">
+        <div class="bg-black rounded-md p-1.5 inline-block -top-8 z-index-0 absolute" v-if="item.code">
```

Comment:
```suggestion
        <div class="bg-black rounded-md p-1.5 inline-block -top-8 z-0 absolute" v-if="item.code">
```
But I'm not sure if it's really needed